utils/apiObjects.ts

```diff
@@ -0,0 +1,151 @@
+import { hexToDecimal, stringToFelt } from "./feltService";
```

Comment:
Having a class for each thing returned by the API is the right thing to do but I would say one class per file, imo it shouldn't be in utils too.

Reply:
I will move it then

Reply:
Done, see: https://github.com/starknet-id/app.starknet.id/pull/580/commits/5f477580a70bc068928fe308ad96eee36cdce6f2